realm/realm-library/src/androidTest/java/io/realm/RealmLinkTests.java

```diff
@@ -572,14 +552,4 @@ public void isNotEmpty() {
         RealmResults<Owner> owners2 = testRealm.where(Owner.class).isNotEmpty("dogs.name").findAll();
         assertEquals(1, owners2.size());
     }
-
-    @Test
-    public void isNullWrongType() {
```

Comment:
Is there a test for nullability of floats if it is now supported? And again maybe note it in the CHANGELOG?

Reply:
We no longer have nullability checks, even the core string parser does not do null checks.